index.bs

```diff
@@ -4892,6 +4892,112 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+## Cloud-Assisted BLE Registration Extension (cable) ## {#sctn-cable-registration-extension}
+
+This [=registration extension=] allows [=[RPS]=] to pass additional data to an
+authenticator that is needed to facilitate caBLE pairing for credential creation.
+
+: Extension identifier
+:: `cableRegistration`
+
+: Client extension input
+:: A Javascript object defined as follows:
+    <xmp class="idl">
+    dictionary CableRegistrationData {
+      required sequence<long> versions;
+      required BufferSource rpPublicKey;
+    };
+
+    partial dictionary AuthenticationExtensionsClientInputs {
+      CableRegistrationData cableRegistration;
+    };
+    </xmp>
+
+    Versions is a list of protocol versions supported by the relying party.
```

Comment:
Could you elaborate the `version` ? What value is supported here?

Reply:
For now only "1", but the field exists so that we can make changes/fixes to the protocol if needed.